src/pages/SignInPage.jsx

```diff
@@ -179,13 +179,15 @@ function SignInPage(props) {
                             type="text"
                             value={username}
                             onChange={(e) => setUsername(e.target.value)}
+                            required
                         />
```

Comment:
```suggestion
                            onChange={(e) => setUsername(e.target.value)}
                            minLength="3"
                            maxLength="20"
                            required
                        />
```
- Conforme limitação da criação, não existem usernames fora destas restrições. Portanto, cabe validar aqui também;